pos_tare/README.rst

```diff
@@ -0,0 +1,122 @@
+====================
+Point Of Sale - Tare
+====================
+
+.. 
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! source digest: sha256:308df37ca0d8288a8593ab10beebb59eb7922a9029c1f976084a2ac5c3d14d12
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fpos-lightgray.png?logo=github
+    :target: https://github.com/OCA/pos/tree/16.0/pos_tare
+    :alt: OCA/pos
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/pos-16-0/pos-16-0-pos_tare
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/builds?repo=OCA/pos&target_branch=16.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+Give the possibility to the user to enter a Tare weight, when weighing
+products in the Point of Sale.
+This will compute automatically net weight and set it to the current
+selected order
+
+.. figure:: https://raw.githubusercontent.com/OCA/pos/16.0/pos_tare/static/description/pos_tare.png
+
+The net weight is displayed in the order with the tare value below.
+
+.. figure:: https://raw.githubusercontent.com/OCA/pos/16.0/pos_tare/static/description/order.png
+
+
+All informations will be available on the receipt.
+
+.. figure:: https://raw.githubusercontent.com/OCA/pos/16.0/pos_tare/static/description/pos_ticket.png
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+Install this add-on and configure your point of sale. To enable this addon, go to your point of sale configuration page. There, enable the electronic scale and barcode reader in the "IoT Box" section. In the same page, look up for the "Tare input method" field, and choose a tare method. There are three tare methods:
+
+   - "manual": you'll set the tare value when you weight the product;
+   - "barcode": you'll scan the tare value from a barcode;
+   - "both": you can both of the above.
+
+.. figure:: https://raw.githubusercontent.com/OCA/pos/16.0/pos_tare/static/description/pos_config.png
+
+
+To generate a tare barcode you need to use the default barcode nomenclature. The default barcode pattern is `0700000{NNDDD}`. Using that pattern, the barcode for a tare of 0.1kg is `0700000001006`. The `pos_tare_barcode_generator` allows you to create tare labels right from the POS.
```

Comment:
```suggestion
To generate a tare barcode you need to use the [default barcode nomenclature](https://www.odoo.com/documentation/16.0/applications/inventory_and_mrp/barcode/operations/barcode_nomenclature.html). The default EAN-13 barcode pattern is `0700000{NNDDD}C`, where `DDD` are the grams, `NN` are the kilograms and `C` is a check digit. Using that pattern, the barcode for a tare of 0.1kg is `0700000001006`. The maximum weight that can be represented with this barcode is just below 100 kg. The `pos_tare_barcode_generator` allows you to create tare labels right from the POS.
```

Reply:
i’m not aware of this module. i quickly looked in earlier versions but did not find it. another module ([``pos_self_service_weighing_tare``](https://github.com/coopiteasy/pos/tree/16.0-self-service-weighing/pos_self_service_weighing_tare), see #1397) allows to do this (but not from another pos than the one used to sell products).